src/api/controllers/study.controllers.js

```
@@ -1 +1,31 @@
 // 요청 파싱(params/query/body) + 입력 검증 결과 처리
+import studyService from '../services/Study.Services.js';
```

ESLint

[error] Unexpected use of file extension "js" for "../services/Study.Services.js"

(import/extensions)